README.md

```diff
@@ -8,22 +8,77 @@
   <br>
 </h1>
 
-# Features
-- hot reloading
-- basic functionality
+This module implements the [`rdk:component:sensor` API](https://docs.viam.com/components/sensor) and provides two sensor models:
 
-# Models
 ```
 zaporter:sds011:v1
 zaporter:sds011:v1-fake
 ```
-# Example Config
+
+The `zaporter:sds011:v1` model supports the SDS011 Nova PM air quality sensor.
+The `zaporter:sds011:v1-fake` model can be used for testing the module without hardware.
+
+[!NOTE]
+> For more information, see [Modular Resources](https://docs.viam.com/registry/#modular-resources).
+
+## Features
+
+- Hot reloading
+- Basic functionality
+
+## Configure your SDS011 sensor
+
+> [!NOTE]
+> Before configuring your sensor, you must [create a machine](https://docs.viam.com/manage/fleet/machines/#add-a-new-machine).
+
+Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com/).
+Click on the **Components** subtab and click **Create component**.
+Select the `sensor` type, then select the `sds011:v1` model.
+Click **Add module**, then enter a name for your sensor and click **Create**.
+
+On the new component panel, copy and paste the following attribute template into your sensor’s **Attributes** box:
+
+{
+  "usb_interface": "<PATH TO USB PORT WHERE YOUR SENSOR IS PLUGGED IN>"
+}
```

Comment:
This isn't rendering correctly. I think this needs to be in triple tick marks

Reply:
Oops yes you're quite right